docs/repo_structure.md

```diff
@@ -0,0 +1,73 @@
+# Repository structure
+
+
+### Rpm-ostree source code
+```
+.
+└─ src                          Contains source code for rpm-ostree
+```
+
+#### File structures under src
```

Comment:
I think we can just collapse these? Using `src/` for source code is pretty universal, so we can skip the explanation for that. :)